### PR TITLE
Migrate shared_governance_base to version 12.0

### PR DESCRIPTION
**Pull request by carmenbianca**
#5 broke when I did some rebasing, so creating a new PR

Task: https://gestion.coopiteasy.be/web#id=7883&view_type=form&model=project.task&action=475&active_id=210

### CODE REVIEW

File: shared_governance_base/views/mandate.xml [hunk 1]
```diff
@@ -90,10 +90,9 @@
     <record id="action_shared_governance_mandate" model="ir.actions.act_window">
         <field name="name">Mandates</field>
         <field name="res_model">sg.mandate</field>
-        <field name="context">{'search_default_state': 'alive'}</field>
+        <field name="context">{'search_default_state': 'alive', 'search_default_alive': 1}</field>
```
Comment (carmenbianca) [inline shared_governance_base/views/mandate.xml]: I have no idea if this is good

File: shared_governance_base/models/mandate.py [hunk 2]
```diff
@@ -1,11 +1,10 @@
-# -*- coding: utf-8 -*-
-from openerp import api, fields, models
+from odoo import fields, models
 
 
 class Mandate(models.Model):
     _name = "sg.mandate"
 
-    _inherit = ["mail.thread", "ir.needaction_mixin"]
+    _inherit = ["mail.thread", "mail.activity.mixin"]
```
Comment (robinkeunen) [inline shared_governance_base/models/mandate.py]: I think we said in #4 to remove `mail.activity.mixin`

File: shared_governance_base/security/shared_governance_security.xml [hunk 3]
```diff
@@ -1,26 +1,27 @@
...
-            <field name="users" eval="[(4, ref('base.user_root'))]"/>
-            <field name="category_id" ref="module_category_shared_governance"/>
-        </record>
-    </data>
+    <record id="group_shared_governance_coordinator" model="res.groups">
+        <field name="name">Coordinator</field>
+        <field name="implied_ids" eval="[(4, ref('group_shared_governance_user'))]" />
+        <field
+            name="users"
+            eval="[(4, ref('base.user_root')), (4, ref('base.user_admin'))]"
```
Comment (robinkeunen) [inline shared_governance_base/security/shared_governance_security.xml]: Only`base.user_admin` is needed here. From 9.0 to 12.0, `base.user_root` was split into `base.user_root` and `base.user_admin`.

**Review by robinkeunen — CHANGES_REQUESTED** [review]

File: shared_governance_base/security/shared_governance_security.xml [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+        <field name="implied_ids" eval="[(4, ref('base.group_user'))]" />
+        <field name="category_id" ref="module_category_shared_governance" />
+    </record>
+
+    <record id="group_shared_governance_coordinator" model="res.groups">
+        <field name="name">Coordinator</field>
+        <field name="implied_ids" eval="[(4, ref('group_shared_governance_user'))]" />
+        <field
+            name="users"
+            eval="[(4, (4, ref('base.user_admin'))]"
```
Comment (huguesdk) [inline shared_governance_base/security/shared_governance_security.xml]: there seems to be a `(4, ` too much. ☺

File: shared_governance_base/models/mandate.py (hunk 2 above)
Comment (carmenbianca) [reply shared_governance_base/models/mandate.py]: Removed.

File: shared_governance_base/security/shared_governance_security.xml [hunk 5]
```diff
@@ -18,10 +18,7 @@
     <record id="group_shared_governance_coordinator" model="res.groups">
         <field name="name">Coordinator</field>
         <field name="implied_ids" eval="[(4, ref('group_shared_governance_user'))]" />
-        <field
-            name="users"
-            eval="[(4, (4, ref('base.user_admin'))]"
-        />
+        <field name="users" eval="[(4, ref('base.user_admin')]" />
```
Comment (huguesdk) [inline shared_governance_base/security/shared_governance_security.xml]: thanks for the change, but there is now a parenthesis missing. ☺

Comment (carmenbianca) [reply shared_governance_base/security/shared_governance_security.xml]: Well this is embarrassing.

Comment (huguesdk) [reply shared_governance_base/security/shared_governance_security.xml]: a good example for ci. 😁

**Pull request merged**